tests/gocase/util/server.go

```diff
@@ -59,6 +59,7 @@ func (s *KvrocksServer) Close() {
 
 func StartServer(t *testing.T, configs map[string]string) *KvrocksServer {
 	b := os.Getenv("KVROCKS_BIN_PATH")
+	require.NotEmpty(t, b, "please set the environment variable `KVROCKS_BIN_PATH`")
```

Comment:
can we make some guesses about the path or just provide a predefined value here, and only fail when we really cannot find the binary?

Reply:
Can you give some guesses here? Actually as suggested in #813 I think basically we can treat Go cases as a standalone module, then we make guesses in `x.py` since there's where we confirm we're in the kvrocks workspace.